charts/cdf/tekton-pipeline/defaults.yaml

```diff
@@ -1,3 +1,3 @@
 gitUrl: https://github.com/cdfoundation/tekton-helm-chart
 namespace: tekton-pipelines
-version: 0.29.0
+version: 0.1.0
```

Comment:
i think this should be 0.2.0, https://github.com/cdfoundation/tekton-helm-chart/blob/main/charts/tekton-pipeline/Chart.yaml#L4

Reply:
So 0.2.0 has one extra change over 0.1.0. I wanted to keep the change exclusive to tekton version upgrade.
Next week we will do 0.2.0